JSONLD/organisations/consortia/solaris-heppa.json

```diff
@@ -0,0 +1,374 @@
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/pmod"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/uobergen"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/uoleeds"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/uomontreal"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/uootago"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2016,
+                    "to": 2022,
+                    "phase": "CMIP6"
+                },
+                {
+                    "from": 2025,
+                    "to": "-",
+                    "phase": "CMIP7"
+                }
+            ]
+        },
+        {
+            "@type": "consortia:member",
+            "consortia:institution": {
+                "@id": "mip-cmor-tables:organisations/institutions/wdcgg"
+            },
+            "consortia:membership_type": "current",
+            "consortia:dates": [
+                {
+                    "from": 2025,
```

Comment:
Keeping track of this will be hard, and to simplify the task, I would omit. Briony and Beth might be good to consult on this TBH?